src/ecs/systems/transform.rs

```diff
@@ -104,7 +105,7 @@ impl System<()> for TransformSystem {
                         // If the index is none then the parent is an orphan or dead
                         if let Some(parent_index) = self.indices.get(&child.parent()) {
```

Comment:
Okay, thanks for checking! I don't have my computer in front of me at the moment, and I'm operating off my phone. 👍